pkg/granted/sso.go

```diff
@@ -41,6 +42,11 @@ var SSOCommand = cli.Command{
 	Subcommands: []*cli.Command{&GenerateCommand, &PopulateCommand, &LoginCommand},
 }
 
+const (
+	// permission for user to read/write/execute.
+	USER_READ_WRITE_PERM = 0700
```

Comment:
The file only needs read write and not execute
```suggestion
	USER_READ_WRITE_PERM = 0644 
```